make/autoconf/flags-ldflags.m4

```diff
@@ -72,6 +72,8 @@ AC_DEFUN([FLAGS_SETUP_LDFLAGS_HELPER],
 
     if test "x$OPENJDK_TARGET_OS" = xlinux; then
       BASIC_LDFLAGS="-fuse-ld=lld -Wl,--exclude-libs,ALL"
+      # Linux/clang toolchain needs now lld on the system to work
+      UTIL_REQUIRE_PROGS(LLD, lld)
```

Comment:
I would suggest a more concise comment and moving it up a line as the parameter forcing the use of lld is right there.

```suggestion
      # Clang needs the lld linker to work correctly
      BASIC_LDFLAGS="-fuse-ld=lld -Wl,--exclude-libs,ALL"
      UTIL_REQUIRE_PROGS(LLD, lld)
```